priam/src/main/java/com/netflix/priam/notification/BackupNotificationMgr.java

```diff
@@ -102,6 +107,42 @@ private Map<String, MessageAttributeValue> getMessageAttributes(
                 new MessageAttributeValue()
                         .withDataType("String")
                         .withStringValue(backupFileType.name()));
+
+        String additionalAttrs =
+                this.backupRestoreConfig.getBackupNotificationAdditionalMessageAttrs();
+        if (!StringUtils.isBlank(additionalAttrs)) {
+            try {
+                List<String> attrs =
+                        Splitter.on(",")
+                                .omitEmptyStrings()
+                                .trimResults()
+                                .splitToList(additionalAttrs);
+                for (String attr : attrs) {
+                    Object value = additionalAttributes.get(attr);
+                    if (value != null) {
+                        // Does not handle all data types
+                        if ("Instant".equals(value.getClass().getSimpleName())) {
```

Comment:
Remove this block as it will never get hit. Note that Instant is not a valid type to put into a JSONObject. 

Per the Javadoc of JSONObject.put(String, Object):

* @param value An object which is the value. It should be of one of these
     *  types: Boolean, Double, Integer, JSONArray, JSONObject, Long, String,
     *  or the JSONObject.NULL object.